Makefile

```diff
@@ -48,5 +49,4 @@ ci: ## run all ci checks
 	make regenerate
 	@git diff --exit-code . || { echo "ERROR: commit and working copy differ after 'make regenerate'"; exit 22 ; }
 	make test
-	golangci-lint run
-	@git diff --exit-code . || { echo "ERROR: commit and working copy differ after 'make ci'" ; exit 2  ; }
+	@git diff --exit-code . || { echo "ERROR: commit and working copy differ after 'make test'" ; exit 22 ; }
```

Comment:
make regenerate is now running make lint, so there's no reason to call it here anymore.